source/fuzz/transformation_record_synonymous_constants.cpp

```diff
@@ -19,8 +19,67 @@ namespace spvtools {
 namespace fuzz {
 
 namespace {
-bool IsScalarZeroConstant(const opt::analysis::Constant* constant) {
-  return constant->AsScalarConstant() && constant->IsZero();
+
+// Returns true if the two given constants are equivalent
+// (the description of IsApplicable specifies the conditions they must satisfy
+// to be considered equivalent)
```

Comment:
Move comment to the .h file - that's the place for top-level method comments.

Reply:
Done. I made the method a static member method and moved the comment to the declaration in the header file.